apps/asn1parse.c

```diff
@@ -20,62 +20,73 @@
+    OPT_INFORM,
+    OPT_IN,
+    OPT_OUT,
+    OPT_INDENT,
+    OPT_NOOUT,
+    OPT_OID,
+    OPT_OFFSET,
+    OPT_LENGTH,
+    OPT_DUMP,
+    OPT_DLIMIT,
+    OPT_STRPARSE,
+    OPT_GENSTR,
+    OPT_GENCONF,
+    OPT_STRICTPEM,
     OPT_ITEM
 } OPTION_CHOICE;
 
 const OPTIONS asn1parse_options[] = {
     OPT_SECTION("General"),
-    {"help", OPT_HELP, '-', "Display this summary"},
-    {"oid", OPT_OID, '<', "file of extra oid definitions"},
+    { "help", OPT_HELP, '-', "Display this summary" },
+    { "oid", OPT_OID, '<', "file of extra oid definitions" },
 
     OPT_SECTION("I/O"),
-    {"inform", OPT_INFORM, 'A', "input format - one of DER PEM B64"},
-    {"in", OPT_IN, '<', "input file"},
-    {"out", OPT_OUT, '>', "output file (output format is always DER)"},
-    {"noout", OPT_NOOUT, 0, "do not produce any output"},
-    {"offset", OPT_OFFSET, 'p', "offset into file"},
-    {"length", OPT_LENGTH, 'p', "length of section in file"},
-    {"strparse", OPT_STRPARSE, 'p',
-     "offset; a series of these can be used to 'dig'"},
-    {OPT_MORE_STR, 0, 0, "into multiple ASN1 blob wrappings"},
-    {"genstr", OPT_GENSTR, 's', "string to generate ASN1 structure from"},
-    {"genconf", OPT_GENCONF, 's', "file to generate ASN1 structure from"},
-    {"strictpem", OPT_STRICTPEM, 0,
-     "equivalent to '-inform pem' (obsolete)"},
-    {"item", OPT_ITEM, 's', "item to parse and print"},
-    {OPT_MORE_STR, 0, 0, "(-inform  will be ignored)"},
+    { "inform", OPT_INFORM, 'A', "input format - one of DER PEM B64" },
+    { "in", OPT_IN, '<', "input file" },
+    { "out", OPT_OUT, '>', "output file (output format is always DER)" },
+    { "noout", OPT_NOOUT, 0, "do not produce any output" },
+    { "offset", OPT_OFFSET, 'p', "offset into file" },
+    { "length", OPT_LENGTH, 'p', "length of section in file" },
+    { "strparse", OPT_STRPARSE, 'p',
+        "offset; a series of these can be used to 'dig'" },
+    { OPT_MORE_STR, 0, 0, "into multiple ASN1 blob wrappings" },
+    { "genstr", OPT_GENSTR, 's', "string to generate ASN1 structure from" },
+    { "genconf", OPT_GENCONF, 's', "file to generate ASN1 structure from" },
+    { "strictpem", OPT_STRICTPEM, 0,
+        "equivalent to '-inform pem' (obsolete)" },
+    { "item", OPT_ITEM, 's', "item to parse and print" },
+    { OPT_MORE_STR, 0, 0, "(-inform  will be ignored)" },
 
     OPT_SECTION("Formatting"),
-    {"i", OPT_INDENT, 0, "indents the output"},
-    {"dump", OPT_DUMP, 0, "unknown data in hex form"},
-    {"dlimit", OPT_DLIMIT, 'p',
-     "dump the first arg bytes of unknown data in hex form"},
-    {NULL}
+    { "i", OPT_INDENT, 0, "indents the output" },
+    { "dump", OPT_DUMP, 0, "unknown data in hex form" },
+    { "dlimit", OPT_DLIMIT, 'p',
+        "dump the first arg bytes of unknown data in hex form" },
+    { NULL }
 };
 
-static int do_generate(char *genstr, const char *genconf, BUF_MEM *buf);
+static int do_generate(char* genstr, const char* genconf, BUF_MEM* buf);
```

Comment:
Is the intention really to reformat the `*` in all pointer declarations?

Reply:
I dont think we should be changing this regardless of what a tool decides to do.

Reply:
It turns out that clang-format does this because it assumes C++. If you read the webkit code style guidelines, you can see that this particular detail differs depending on if the language is C or C++